scripts/daily.js

```diff
@@ -82,7 +82,7 @@ Total: ${powerballPrize + doublePlayPrize + megaPrize}`;
       // incr powerball prize this week
       const powerballPlays = await kv.get('powerball_plays');
       const prize = powerballPlays.reduce((sum, play) => (
-        sum + powerball.calculatePrize(powerballWinningNumbers, play)
+        sum + powerball.checkNumbersForPrize(powerballWinningNumbers, play).prize
```

Comment:
is this working without return?
like 
return sum + powerball.checkNumbersForPrize(powerballWinningNumbers, play).prize ?

Reply:
Thats arrow function Implicit Return.  if you only have one line in your function which is returning something you can put it on one line without the return statement. 
```
function randomNumber() {
  return Math.random()
}
```

```
const randomNumber = () => Math.random()
```